### ISSUE TITLE
Summon the bot when using play command - Version 1.9.7

### ISSUE THREAD

**Issue by Sxcret** [Just-Some-Bots/MusicBot#1638]
- [x] I am using Python 3.5 or higher (run `python --version` on the command line)
- [x] I have followed the [official guides](https://github.com/Just-Some-Bots/MusicBot/wiki) to install the bot for my system
- [x] I have updated my dependencies to the latest version using the appropriate `update` script

Which version are you using?

- [x] The latest `master` version (1.9.7)
- [ ] The latest `review` version

What type of issue are you creating?

- [ ] Bug
- [x] Feature request
- [x] Question

----

### Description
I attempt to make my bot automaticaly joining the channel when user use the command play, without need to do the command summon before.

On the review version it works but I used the version 1.9.7 because it's the last version, it's more cool and spotify work with.

I not very good in coding so I try to modify the code by myself but it doesn't work, so I need your help.

----

### Log file

[musicbot.log](https://github.com/Just-Some-Bots/MusicBot/files/2075020/musicbot.log)

----

Thank you very much,
Sxcret

**Comment by AutumnClove** [comment]
The bot is open source and licensed under MIT which means you can modify it, but we’re not here to give you help with doing so. There's a server for help setting up and running the bot. We will not help you with modifying it or with errors that occur by modifying it however.

**Comment by Sxcret** [comment]
#837 And this ?

It's also like a feature request, because I think that lot of people want to use only one command, to make the bot join the channel and play their link.

**Comment by AleanaA** [comment]
Instead of just referencing an issue and saying "What about this?" please go through that issue and actually take a look at what was said in the issue. It has information as to how to do it.

Whether we help you do it or not, this is a duplicate of 837 and can be closed.

**Comment by Sxcret** [comment]
"On the review version it works but I used the version 1.9.7 because it's the last version, it's more cool and spotify work with."

Please read before comment... I referenced this issue that means that I have already attempt to use it...

**Comment by AleanaA** [comment]
That thread is from before 1.9.7 was released, not that it matters, the explanations still apply here.

Going by a quick read, you clearly didn't read enough of the thread to attempt to do anything actually recommended.

> you take the code from the summon command, adapt it, and put it at the top of the play command

**Comment by Sxcret** [comment]
"I not very good in coding so I try to modify the code by myself but it doesn't work, so I need your help."

**Comment by AleanaA** [comment]
> The bot is open source and licensed under MIT which means you can modify it, but we’re not here to give you help with doing so.

If you can't figure out how to copy and paste something and tweak it just slightly to make it work, you probably shouldn't be modifying the bot imho. To quote the same person from that thread

> spoonfeeding is bad, I vow not to spoonfeed

**Comment by Sxcret** [comment]
I request help because I attempt to modify the code for upgrade the bot and set it more easy to use. I try lot of things to summon the bot when the play command is set but I didn't succeed. That's why I came here to find help... But I see that you prefer to answer absurd things rather than simply help me, you waste your time as you lose mine. So if it's just for a moral issue, I'll wait for answers from people who really want to help me.
Everyone are not expert in python, so help only once a thing you already know how to do, I do not see the problem and you will avoid having to answer again and again to this kind of request.

**Comment by AutumnClove** [comment]
We *don't help* with coding is what @AleanaA and I are trying to commute. Where as we are competent users and above in the support server.

**Comment by AleanaA** [comment]
> you take the code from the summon command, adapt it, and put it at the top of the play command

This is literally what you have to do. The code is all there, all you have to do is move it from one place to the other. If you'd like to tell me that explaining step by step how to copy and paste a set of lines is not 'spoonfeeding' as they put it, please, by all means, inform me.

This isn't the place to ask for help for help regarding modifying the bot. This is the issues section to request help for issues with the bot, feature requests, and questions regarding what the bot can do.

I return to my previous statement. This thread can be closed as this is a duplicate of 837

**Comment by Sxcret** [comment]
"This isn't the place to ask for help for help regarding modifying the bot. This is the issues section to request help for issues with the bot, feature requests, and questions regarding what the bot can do."

That's because I said that that can be a Feature, because I think that lot of people didn't want the summon command before play.

Thanks for your help

**Comment by ImthePeach** [comment]
#837
